src/components/FilterHeader.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext, useEffect } from 'react';
```

Comment:
Can we rename this component to make it clear it's a Capture filter header?
We could rename `FilterTop` at the same time.